src/prefect/server/database/orm_models.py

```diff
@@ -833,6 +833,21 @@ def __table_args__(cls):
         )
 
 
+@declarative_mixin
+class ORMDeploymentSchedule:
+    @declared_attr
+    def deployment_id(cls):
+        return sa.Column(
+            UUID(),
+            sa.ForeignKey("deployment.id", ondelete="CASCADE"),
+            nullable=False,
+            index=True,
+        )
+
+    schedule = sa.Column(Pydantic(schemas.schedules.SCHEDULE_TYPES), nullable=False)
+    active = sa.Column(sa.Boolean, nullable=False, default=True)
```

Comment:
should it be `is_active`?

Reply:
The language I've been using is that a deployment can be 'paused' but a schedule is active or inactive. I think this is a good approach so we're not conflating terminology. If both were 'paused' then saying a Deployment is paused could have multiple meanings.

Reply:
mm I was actually just saying `is_active` instead of `active`, but if we don't change to `is_paused` this comment is a no-op anyway